tf2_ros/src/buffer_server_main.cpp

```diff
@@ -76,13 +62,10 @@ int main(int argc, char** argv)
     node_name = "tf2_buffer_server";
   }
 
-  // WIM: this works fine:
```

Comment:
How is this change related to the above unused code? How have you verified that this change is appropriate?

Reply:
See my long comment.